### DISCUSSION TITLE
Is it legal to use this API for commercial projects

### DISCUSSION THREAD

**Discussion by Joe02exe** [userbradley/BeReal#86] [Q&A] [answered]
Hello,

I find the documentation of your project very informative, especially regarding the API and other aspects. However, I have a question regarding the legality of using this API without BeReal's knowledge. To simplify my question: "Is it legal to use third-party embeds (i.e., the API to retrieve the latest BeReals) for other projects or even for commercial purposes?"

Thank you.

**Comment by RihanArfan** [comment, marked as answer]
If it's a commercial project, I don't think so. You should expect any API route to break at any moment, as it's not officially available for use outside of BeReal's apps.

**Comment by userbradley** [comment]
Nope, this is provided as an educational asset and you should assume (like @RihanArfan said) that the API will break at any moment.
